### PR TITLE
CVs Page

### PR DESCRIPTION
**Pull request by lnicepei**
Created CVs page with the ability to create, assign and unbind cvs
(Yup, too many lines of code added. I'm working on reducing it)
Here is a demo
https://cv-builder-g6313dosf-cv-builder.vercel.app
image

### CODE REVIEW

File: src/components/NewResumeModal/hooks/useOptions.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { useLazyQuery } from '@apollo/client'
+
+export const useOptions = () => {
+  const [options, setOptions] = useState<{
+    languages: Language[]
+    skills: Skill[]
+    projects: Project[]
+  }>()
+  const [optionsQuery] = useLazyQuery(GET_LANGUAGES_AND_SKILLS_AND_PROJECTS)
+  //should it be like so??
```
Comment (Ascilator) [inline src/components/NewResumeModal/hooks/useOptions.ts:13]: why not?

File: src/components/NewResumeModal/types.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export interface NewResumeFormValues {
+  languages: {
+    language: [string, string]
+  }[]
```
Comment (Ascilator) [inline src/components/NewResumeModal/types.ts:4]: bad syntax
it suppose to be Array<type>

File: src/pages/EmployeePage/EmployeePage.style.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { Spin } from 'antd'
-import { SpinFCType } from 'antd/lib/spin'
+
```
Comment (Ascilator) [inline src/pages/EmployeePage/EmployeePage.style.ts:2]: mmmmmm break, let it be :)

File: src/pages/EmployeePage/components/Languages/Languages.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+type LanguagesProps = {
+  languagesData: Language[]
+}
+
+const Languages: React.FC<LanguagesProps> = ({ languagesData }) => {
+  const langs = languagesData.map((l) => <div key={l.id}>{l.language_name}</div>)
```
Comment (Ascilator) [inline src/pages/EmployeePage/components/Languages/Languages.tsx:8]: letter l - so meaningful

File: src/pages/EmployeePage/components/Profile/Profile.tsx [hunk 5]
```diff
@@ -1,49 +1,51 @@
...
+}) => {
+  const { updateEmployee, isFetching: isUpdateFetching } = useUpdateEmployee()
+  const { departments, positions, isFetching: areOptionsFetching } = useOptions()
 
   const initialValues = {
-    first_name: employee?.profile.first_name,
-    last_name: employee?.profile.last_name,
-    positionId: employee?.position?.id,
-    departmentId: employee?.department?.id
+    first_name: firstName,
```
Comment (Ascilator) [inline src/pages/EmployeePage/components/Profile/Profile.tsx:31]: why not first_name or firstName in both cases?

File: src/pages/EmployeesPage/components/UpdateEmployeeModal/UpdateEmployeeModal.tsx [hunk 6]
```diff
@@ -1,11 +1,16 @@
 import React from 'react'
 
-import { useAppSelector } from '@/state'
 import { Modal } from 'antd'
+
+import { useOptions } from '@/components/UpdateEmployeeForm/hooks/useOptions'
+import { UpdateEmployeeFormValues } from '@/components/UpdateEmployeeForm/types'
 import UpdateEmployeeForm from '../../../../components/UpdateEmployeeForm/UpdateEmployeeForm'
+//is it okay to import from such a faraway folder?
```
Comment (Ascilator) [inline src/pages/EmployeesPage/components/UpdateEmployeeModal/UpdateEmployeeModal.tsx:8]: nope)

**Review by Ascilator — COMMENTED** [review]
everything above

File: src/pages/EmployeePage/components/Languages/Languages.tsx (hunk 4 above)
Comment (lnicepei) [reply src/pages/EmployeePage/components/Languages/Languages.tsx:8]: I will fix it as soon as I start developing this part of functionality, but I got your point

File: src/pages/EmployeePage/EmployeePage.style.ts (hunk 3 above)
Comment (lnicepei) [reply src/pages/EmployeePage/EmployeePage.style.ts:2]: break? what do you mean?

**Pull request merged**